### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by davidask**

### CODE REVIEW

File: Sources/SQL/Core/RowProtocol.swift [hunk 1]
```diff
@@ -114,27 +114,38 @@ public extension RowProtocol {
...
     public func value<T: ValueConvertible>(_ field: Table.Field) throws -> T {
-        return try row.value(Table.field(field))
+        return try value(Table.field(field))
     }
     
     public func value<T: ValueConvertible>(_ field: Table.Field) throws -> T? {
-        return try row.value(Table.field(field))
+        return try value(Table.field(field))
     }
 }
```
Comment (Danappelxx) [inline Sources/SQL/Core/RowProtocol.swift]: empty line at EOF please :)

**Comment by antonmes** [comment]
😱 This should be named "a complete rewrite", hahah kidding 😂. awesome, man!

**Comment by Danappelxx** [comment]
I think we're just about ready to merge!

**Comment by davidask** [comment]
Let me check PostgreSQL. I'll merge later today! Great work!

**Pull request merged**